tokio/src/sync/watch.rs

```diff
@@ -644,8 +644,8 @@ impl<T> Receiver<T> {
         Ok(self.version != new_version)
     }
 
-    /// Marks the state as unseen.
-    pub fn mark_unseen(&mut self) {
+    /// Marks the state as changed.
+    pub fn mark_changed(&mut self) {
```

Comment:
This documentation is insufficient.

That said, this should probably have been done in the previous PR. Do you want to update the docs here? Otherwise I'll open an issue for it and merge this.

Reply:
Not sure what is expected here. I have added some docs including a possible use case.